packages/flutter_adaptive_scaffold/lib/src/adaptive_scaffold.dart

```diff
@@ -317,13 +317,10 @@ class AdaptiveScaffold extends StatefulWidget {
   }) {
     return Builder(
       builder: (_) {
-        return BottomNavigationBar(
-          currentIndex: currentIndex ?? 0,
-          iconSize: iconSize,
```

Comment:
It looks like the updated code ignores `iconSize` now. Could you reintroduce it with the `NavigationBarThemeData.iconTheme`?

Reply:
added